artifactory/services/utils/aqlquerybuilder.go

```diff
@@ -13,10 +14,13 @@ import (
 // Returns an AQL body string to search file in Artifactory by pattern, according the specified arguments requirements.
 func CreateAqlBodyForSpecWithPattern(params *CommonParams) (string, error) {
 	searchPattern := prepareSourceSearchPattern(params.Pattern, params.Target, true)
-	repoPathFileTriples, err := createRepoPathFileTriples(searchPattern, params.Recursive)
+	repoPathFileTriples, singleRepo, err := createRepoPathFileTriples(searchPattern, params.Recursive)
 	if err != nil {
 		return "", err
 	}
+	if params.Transitive && !singleRepo {
+		return "", errorutils.CheckErrorf("Repository name cannot contain wildcards in transitive search.")
```

Comment:
Repository name cannot contain wildcards in transitive search.
-->
When searching or downloading with the `transitive` option, the pattern must include a single repository only, meaning wildcards are allowed only after the first slash